pkg/cache/v3/linear.go

```diff
@@ -149,6 +154,8 @@ type LinearCache struct {
 
 	watchCount int
 
+	customWildcardMode bool
```

Comment:
`customWildcardMode` does not describe the behavior here
The name should describe what is happening in this case
Also other behavior altering booleans in the Cache use a `useStg` pattern